testcontainers/src/test/java/io/debezium/examples/testcontainers/DebeziumContainerTest.java

```diff
@@ -48,17 +49,17 @@ public class DebeziumContainerTest {
     private static KafkaContainer kafkaContainer = new KafkaContainer()
             .withNetwork(network);
 
-    public static PostgreSQLContainer<?> postgresContainer = new PostgreSQLContainer<>("debezium/postgres:11")
+    public static PostgreSQLContainer<?> postgresContainer = new PostgreSQLContainer<>(DockerImageName.parse("debezium/postgres:11").asCompatibleSubstituteFor("postgres"))
             .withNetwork(network)
             .withNetworkAliases("postgres");
 
-    public static DebeziumContainer debeziumContainer = new DebeziumContainer(System.getProperty("debeziumVersion"))
+    public static DebeziumContainer debeziumContainer = new DebeziumContainer("debezium/connect:" + System.getProperty("debeziumVersion"))
```

Comment:
Where's the value for that property coming from? Can't we just use `DebeziumContainer.latestStable()`, as in other tests?

Reply:
The value comes from the surefire plugin's configuration to pass `version.debezium` from the pom.

Reply:
Ok, but this makes it harder to run this test in an IDE. I'd suggest to use `latestStable()` for ease of use.

Reply:
Done